Final_Project/Project_Proposal.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "GEO 511 Project Proposal"
+author: "Collin O'Connor"
+date: "10/14/2020"
+output: github_document
+---
+
+<style type="text/css">
+h1.title {text-align: center;}
+h4.author {text-align: center;}
+h4.date {text-align: center;}
+</style>
+
+
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Problem / Question:
+
+The deer tick or blacklegged tick (<i>Ixodes scapularis</i>) is the most medically relevant species of hard tick that in New York State (NYS). <i>I. scapularis</i> populations in NYS may carry a plethora of pathogens, including but not limited to:
+
+* <i>Borrelia burgdorferi</i> - the causitive agent of Lyme disease
+* <i>Anaplasma phagocytophilum</i> - the causitive agent of human granulocytic anaplasmosis
+* <i>Babesia microti</i> - the causitive agent of babesiosis in humans
+* <i>Borrelia miyamotoi</i> - causative of a relapsing tick fever 
+
+The prevalence of the above pathogens varies both spatially and temporally in NYS, which results in varying risk for NYS citizens. Because these pathogens are not transmitted from tick to tick transovarially, they are required to be maintained throughout populations of animal hosts in the various ecosystems of NYS. Data suggests that certain land use and land cover types are beneficial for populations of animals which maintain these pathogens, and potentially for the pathogen's themselves. This project aims to explore any trends between pathogen prevalence within <i>I. scapularis</i> at the county level and land use / land cover types in NYS.
```

Comment:
great overview.  In markdown you can also use `_text_` for italics instead of the HTML tags `<i>`